### ISSUE TITLE
Implement an image service for ciao with Open Stack compatibility

### ISSUE THREAD

**Issue by kaccardi** [01org/ciao#383]
Implement the Open Stack compatibility service interface for the image service.

**Comment by kaccardi** [comment]
Implement storage image interface (dependent on Openstack compatibility interface)
Backend for image service will just be cephfs, so we don’t need anything special for a backend driver, just file operations will do.
Define datastore interface (needs to be defined before creating it)

**Comment by obedmr** [comment]
@kaccardi, is this already addressed on this code: https://github.com/01org/ciao/blob/master/openstack/image/api.go
If so, we could close it.

**Comment by markdryan** [comment]
I've tested the image service in Single VM (using #664) with cephfs and it seems to work.  The verify.sh tests are passing at any rate.  What else needs to be done here?  The lack of persistence is covered by #632.  I did notice that not all the ciao-cli image add parameters are handled by the image service, although we could open a separate bug for this, as I've already done for other image-service issues I've found.
